### DISCUSSION TITLE
Error for  Porous Flow Tutorial Page 01. A single fluid

### DISCUSSION THREAD

**Discussion by caoductoan379** [idaholab/moose#28082] [Q&A Modules: Porous Flow]
Hi All. 

I am starting to learn an example of Porous Flow Tutorial Page 01. A single fluid. I am using an HPC cluster. Unfortunately, i got an error as an attachment. 

Could you please let me know what is wrong with this problem? Thank you very much for this help. 

Please see my input file and the error from the screenshot. 

[Fluid1.txt](https://github.com/user-attachments/files/16131403/Fluid1.txt)
![Screenshot 2024-07-08 112453](https://github.com/idaholab/moose/assets/73046057/3f9106a9-52da-4908-84ef-496bc79cfc43)

**Comment by GiudGiud** [comment]
Hello

You are using an object which is not registered to this application. Which executable are you using?

This screenshot is nearly unreadable in dark mode. Please copy paste screen text in triple quotes next time. 

Guillaume

**Reply by caoductoan379** [reply]
Thanks for the quick response. I am using an execution as: 
/modules/solid_mechanics/solid_mechanics-opt –i  input file.

Thanks a lot for this help

**Reply by GiudGiud** [reply]
But this is the porous flow tutorial. So you need to use the porous flow executable, in modules/porous_flow.

You can build it the same way you built the solid mechanics executable

**Reply by caoductoan379** [reply]
Thanks a lot. It is working.
